README.md

```diff
@@ -31,21 +31,23 @@ jobs:
       with:
         creds: ${{ secrets.AZURE_CREDENTIALS }}
 
-    - name: 'Sync NGINX configuration to NGINX on Azure instance'
+    - name: 'Sync the NGINX configuration from the Git repository to the NGINX for Azure deployment'
       uses: nginxinc/nginx-for-azure-deploy-action@v1
       with:
         subscription-id: ${{ secrets.AZURE_SUBSCRIPTION_ID }}
         resource-group-name: ${{ secrets.AZURE_RESOURCE_GROUP_NAME }}
         nginx-deployment-name: ${{ secrets.NGINX_DEPLOYMENT_NAME }}
-        nginx-config-relative-file-path: ${{ secrets.NGINX_CONFIG_FILE }}
+        nginx-config-directory-path: config
```

Comment:
In your [absolute-paths example repo](https://github.com/bangbingsyb/nginx-config-github-composite-action/blob/main/.github/workflows/nginx-multi-files-config-absolute-paths.yml), I see:
```
nginx-config-directory-path: config-absolute-paths
```
Not sure what is expected here - `config` or `config-absolute-paths`